dev/TeachingTip/TeachingTip.xaml

```diff
@@ -581,22 +585,26 @@
                                                         FontStretch="{TemplateBinding FontStretch}"/>
                                                 <Grid Grid.Row="2">
                                                     <Grid.ColumnDefinitions>
-                                                        <ColumnDefinition Width="*"/>
+                                                        <ColumnDefinition x:Name="ActionButtonColumn" Width="*"/>
```

Comment:
I think this can be done in a cleaner way if you set the middle column to the width of the spacing between the buttons, and collapse it when only one of them is present., instead of two margins. Then move the only one button to the correct column.

Reply:
Switched to a spacing column now and introduced the appropriate lightweight styling resources.